entity-framework/core/miscellaneous/cli/dbcontext-creation.md

```diff
@@ -35,8 +35,8 @@ You can also tell the tools how to create your DbContext by implementing the `ID
 [!code-csharp[Main](../../../../samples/core/Miscellaneous/CommandLine/BloggingContextFactory.cs)]
 
 > [!NOTE]
-> The `args` parameter is currently unused. There is [an issue][8] tracking the ability to specify design-time arguments
-> from the tools.
+> Prior to 5.0.0 the `args` parameter was unused (see [this issue][8]). This was fixed in the 5.0.0 release and now any 
```

Comment:
Is "This was fixed in the 5.0.0 release" the wording we are using for these things? I would prefer something like, "This is fixed in EF Core 5.0." since it doesn't imply a release has happened and includes explicit what release we mean--i.e. EF Core 5.0, not .NET 5.